File: Simulation/1945.Sum of Digits of String After Convert/Solution.java
class Solution {
    public int getLucky(String s, int k) {
        StringBuilder sb = new StringBuilder();
        for (char c : s.toCharArray()) {
            sb.append(c - 'a' + 1);
        }
        int ans = 0;
        String ss = sb.toString();
        while (k-- > 0) {
            int cur = 0;
            for (char c : ss.toCharArray()) {
                cur += c - '0';
            }
            ans = cur;
            ss = String.valueOf(cur);
        }
        return ans;
    }
}

File: Simulation/1945.Sum of Digits of String After Convert/Solution.py
class Solution:
    def getLucky(self, s: str, k: int) -> int:
        ss = ""
        for c in s:
            ss += str(ord(c) - ord('a') + 1)
        ans = 0
        while k > 0:
            cur = 0
            for c in ss:
                cur += ord(c) - ord('0')
            ans = cur
            ss = str(cur)
            k -= 1
        return ans

